src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Linalg/SparseVectorUDTTests.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.ML.Linalg;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.IpcTests.ML.Linalg
+{
+    [Collection("Spark E2E Tests")]
+    public class SparseVectorUDTTests
```

Comment:
Can we combine this with `SparseVectorTests` and just add `TestSparseVectorUDT`?